pylint/extensions/check_elif.py

```diff
@@ -31,7 +31,8 @@ class ElseifUsedChecker(BaseTokenChecker):
     name = "else_if_used"
     msgs = {
         "R5501": (
-            'Consider using "elif" instead of "else if"',
+            'Consider using "elif" instead of "else", replace the "else" by an "elif",'
+            ' then de-indent the code and switch from "if" to "elif" inside the else',
```

Comment:
I find this suggestion hard to follow. If I try to follow it on the first example in the unittests, I get something like this.

source
```python
    if myint > 5:
        pass
    else:
        if myint <= 5:  # [else-if-used]
            pass
        else:
            myint = 3
            if myint > 2:
                if myint > 3:
                    pass
```

"replace the "else" by an "elif"...
```python
    if myint > 5:
        pass
    elif myint <= 5:
            pass
        else:
            myint = 3
            if myint > 2:
                if myint > 3:
                    pass
```

" then de-indent the code..."

```python
    if myint > 5:
        pass
    elif myint <= 5
            pass
    else:
        myint = 3
        if myint > 2:
            if myint > 3:
                pass
```

" and switch from "if" to "elif" inside the else"
```python
    if myint > 5:
        pass
    elif myint <= 5
            pass  <-- wait, wasn't this to be unindented?
    else: <-- turn this to elif?
        myint = 3
        if myint > 2:  <-- turn this to elif?
            if myint > 3:
                pass
```

I appreciate what we want to do here to help folks out, but I think it might not do any good in some cases. Sorry for going through it so step by step but just wanted to show if we're trying to help folks maybe less is more.

Reply:
Ha, I made a slight change before seeing this review, let me try to do it step by step again.

Reply:
> remove the ""else"", then de-indent the code and switch from ""if"" to ""elif"" inside the else"

```python
   if myint > 5:
        pass
    else:
        if myint <= 5:  # [else-if-used]
            pass
        else:
            myint = 3
            if myint > 2:
                if myint > 3:
                    pass
```     
 remove the ""else"",

```python
   if myint > 5:
        pass
        if myint <= 5:  # [else-if-used]
            pass
        else:
            myint = 3
            if myint > 2:
                if myint > 3:
                    pass
``` 
 then de-indent the code 
```python
   if myint > 5:
        pass
    if myint <= 5:  # [else-if-used]
        pass
    else:
        myint = 3
        if myint > 2:
            if myint > 3:
                pass
```      
switch from ""if"" to ""elif"" inside the else"
```python
   if myint > 5:
        pass
  elif myint <= 5:
        pass
  else:
        myint = 3
        if myint > 2:
            if myint > 3:
                pass
```         
Seems to work, but then again I'm the one that wrote the message so of course I understand it correctly 😄

Reply:
Yeah, this is better, but it's still hard because "switch from "if" to "elif" inside the else" is referring to an else that's already gone.

What about:
`consider using elif instead of else to remove one indentation level`

Reply:
Yeap, that's nicer let's use this.

Reply:
Maybe:
Consider consolidating if after else to elif

Reply:
I don't understand "consolidating" so I went with ``'Consider using "elif" instead of "else" then "if" to remove one indentation level'``, which is nice and small imo.